server/datastore/mysql/scripts.go

```diff
@@ -96,12 +96,31 @@ func (ds *Datastore) ListPendingHostScriptExecutions(ctx context.Context, hostID
     host_script_results
   WHERE
     host_id = ? AND
-    exit_code IS NULL AND
-    created_at >= DATE_SUB(NOW(), INTERVAL ? SECOND)`
+    exit_code IS NULL
+  ORDER BY
+    created_at ASC`
```

Comment:
just thinking out loud: there's no time filter anymore, but there's an implicit limit of `1000` entries since that's the maximum number of pending scripts we allow. Is that fine? seems so